### DISCUSSION TITLE
Add searchbox for Log browser "Select labels to search in" in explore

### DISCUSSION THREAD

**Discussion by Mario-Hofstaetter** [grafana/grafana#48583] [Dashboards, Panels & Visualizations]
When using the Grafana `8.5.0` explore feature "Metrics browser" for prometheus, it is possible to search for label names (not their values) in the UI by text-search, **which is very handy if you have lots of labels.**

When using the "Log browser" for Loki, this feature is missing. It is only possible to search label _values_, not label _names_.

Please also enable search label names in the Log browser to unify the UX experience 👍 

Link for [play.grafana.org](https://play.grafana.org/explore?orgId=1&left=%7B%22datasource%22:%22demo.robustperception.io%22,%22queries%22:%5B%7B%22refId%22:%22A%22%7D%5D,%22range%22:%7B%22from%22:%22now-6h%22,%22to%22:%22now%22%7D%7D&right=%7B%22datasource%22:%22Loki%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22expr%22:%22%22,%22queryType%22:%22range%22%7D%5D,%22range%22:%7B%22from%22:%22now-6h%22,%22to%22:%22now%22%7D%7D)

![grafik](https://user-images.githubusercontent.com/33002073/166242491-4a34ad93-3849-4739-9119-282d7775cacf.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
